2.6_testers2.ipynb

```diff
@@ -192,8 +229,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## EnqueueSeq and DequeueSeq \n",
-    "Note the usage of `c.in.enqueueSeq` and `c.out.expectDequeueSeq` after some initial boiler plate to set up the decoupled interfaces."
+    "## EnqueueNow and expectDequeueNow\n",
+    "*testers2* has some built in methods for dealing with circuits with Decoupled interfaces in the IOs. In this example we will see how to insert and extract values from the `queue`. \n",
```

Comment:
I don't know if it's discussed elsewhere but it would be nice to rephrase this to not suggest that this is special case logic - it's a library that anyone can write, which I think is one of the main strengths. Perhaps a link to the GitHub page that shows their implementation?

Reply:
Done